projects/plugins/jetpack/modules/widgets/image-widget.php

```diff
@@ -71,23 +75,23 @@ public function widget( $args, $instance ) {
 			echo $args['before_title'] . $title . $args['after_title']; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
 		}
 
-		if ( '' != $instance['img_url'] ) {
+		if ( $instance['img_url'] ) {
 
 			$output = '<img src="' . esc_url( $instance['img_url'] ) . '" ';
 
-			if ( '' != $instance['alt_text'] ) {
+			if ( $instance['alt_text'] ) {
```

Comment:
If the 'alt_text' happens to be the string '0' or the number 0, this change will stop displaying it. Same for the rest of the comparison changes here.

You could preserve the old behavior with something like `'' !== (string) $instance['alt_text']` instead. Or maybe we don't care.

Reply:
Thanks for pointing that out, I don't think it hurts to change like you mentioned e52752a91f05255fad370cb32938d33343b303e3